index.html

```diff
@@ -1345,7 +1345,7 @@ <h2>Definition of Roles</h2>
 			<div class="role-description">
 				<p>A <rref>landmark</rref> that contains mostly site-oriented content, rather than page-specific content.</p>
 				<p>Site-oriented content typically includes things such as the logo or identity of the site sponsor, and a site-specific search tool. A banner usually appears at the top of the page and typically spans the full width.</p>
-				<p>User agents SHOULD treat elements with the role of <code>banner</code> as navigational <a>landmarks</a>.</p>
+				<p><a>Assistive technologies</a> SHOULD enable users to quickly navigate to elements with role <code>banner</code>. Mainstream <a>user agents</a> SHOULD enable users to quickly navigate to <a>elements</a> that have an implicit role of <code>banner</code>.</p>
```

Comment:
Why do we need the word "mainstream"? I also don't understand why a user agent would limit the capability to elements with an implicit role. I have a hard time believing there would be a significant difference in implementation difficulty.


```suggestion
				<p><a>User agents</a> and <a>assistive technologies</a> SHOULD enable users to quickly navigate to elements with role <code>banner</code>.</p>
```

Reply:
Agreed.